doc/ref_model/chapters/chapter09.md

```diff
@@ -214,6 +214,31 @@ All phases described above can be automated using technology specific toolsets a
 <a name="9.5.3.1"></a>
 #### 9.5.3.1. Pre-tenant Creation Requirements
 
+Topics include:
+1. Tenant Approval -- use, capacity, data centers, ..
+    - Validate that the [Tenant's](../../../common/glossary.md#operational-and-administrative-terminology) planned use meets the Operators Cloud Use policies
+    - Validate that the capacity available within the requests cloud site(s) can satisfy the Tenant requested quota for vCPU, RAM, Disk, Network Bandwidth
+    - Validate that the Cloud Infrastructure can meet Tenant's performance requirements (e.g. I/O, latency, jitter, etc)
+    - Validate that the Cloud Infrastructure can meet Tenant's resilience requirements
+1. For VM-based environments:
```

Comment:
We should either add a similar bullet for containerized usage or leave this part for the RAs.

Reply:
@ulikleber The section is for private flavours. Is there something similar for containers?

Reply:
@ulikleber Kubernetes doesn't have the concept of flavors -- changed wording to make it independent of VM